open-source.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Minimum Open Source requirements for Services
+description: The minimum bar for Open Source when working on Services, that then allows further progression.
+---
+
+# Minimum Open Source Requirements for Services
+
+Here we document the minimum bar for use of Open Source in services. When a service meets this minimum bar, it is then possible for contributors to further progress the service toward the complete Operate First methodology. This represents what we expect from any service claiming to be Open Source.
+
+## Requirements
+
+These requirements do not represent best practices, but are the bare minimum for Open Source development of services. Most Open Source models (beyond “throw it over the wall” behavior) are covered by these requirements in one way or another:
+
+ 1. **All code and assets necessary to operate the service are shared under an open source license, and publicly accessible.**
+ 1. **A public contributor can use the same workflow as a typical team member to make a change to the service.**
+
+For many projects the above seem obvious; this is how we generally work together on Open Source projects. Nevertheless such requirements are being explicitly specified, since services are a new area of work, and since the services and their sources are often not “distributed” for users to operate, and optionally modify, their own copy.
+
+## Frequently Asked Questions:
+
+Due to the variety of services, projects, teams and workflows there is a lot of flexibility in the requirements. Secondly, the requirements are a bare minimum, and best practices usually go beyond the requirements. Thus we must address common questions about the requirements:
+
+### Q: What “assets” should be Open Source?
+
+**A:** Make everything open by default, except where law, security, privacy or common sense says otherwise. A general rule of thumb is if you commit it to a repository, it should be open. Assets that should not be shared, are ones where you don’t have redistribution rights (such as a RHEL VM image, or private information, personal identifiable information), or things that are actually secret: Don’t share keys, passwords or tokens.
+
+If changing configuration is an activity that a typical team member does in order to change the service, then you should share that configuration publicly, in order to meet **requirement #2**.
+
+It is *recommended* that you include sufficient configuration to operate the service, or some base default version of configuration. Putting secrets in your configuration is a bad practice that should be avoided, regardless of Open Source.
+
+### Q: Should tests be shared with the code?
+
+**A:** Tests that are used during operation the service (as per **requirement #1**), and tests that used when run when making a change to the service (as per **requirement #2**), *must be shared publicly* under an Open Source license.
```

Comment:
```suggestion
**A:** Tests that are used during operation the service (as per **requirement #1**), and tests that used when run when making a change to the service (as per **requirement #2**), _must be shared publicly_ under an Open Source license.
```
The to-be-published markup standards differentiate italic emphasis with the underscores so it is easier to see the difference in the unrendered MarkDown file, e.g. in a plain text editor that does not show MarkDown styling inline while editing.

Reply:
Updated to use underscore for italics.